protractor.conf.js

```diff
@@ -0,0 +1,30 @@
+const webpackConfig = require('./webpack.config');
+
+exports.config = {
+	framework: "jasmine",
+	baseUrl: `http://localhost:3000`,
+	seleniumAddress: 'http://localhost:4444/wd/hub',
+	specs: [
+		'client/**/**.e2e.js',
+    'client/**/*.e2e.js'
+	],
+	jasmineNodeOpts: {
+    showTiming: true,
+    showColors: true,
+    isVerbose: true,
+    includeStackTrace: true,
+    defaultTimeoutInterval: 400000
+  },
+  directConnect: true,
+  capabilities: {
+    'browserName': 'chrome',
+    'chromeOptions': {
+      'args': ['show-fps-counter=true']
+    }
+  },
+	allScriptsTimeout: 30000,
+	onPrepare: function () {
+    require("babel-core/register")({ retainLines: true });
+    browser.ignoreSynchronization = true;
```

Comment:
This is not best practice and should be avoided. If you want to ignore synchronization, do it on a test-by-test basis rather than app wide.